vulnerabilities/importers/ubuntu.py

```diff
@@ -53,22 +54,22 @@ def __init__(self, *args, **kwargs):
         # we could avoid setting translations, and have it
         # set by default in the OvalParser, but we don't yet know
         # whether all OVAL providers use the same format
-        self.translations = {'less than': '<'}
+        self.translations = {"less than": "<"}
```

Comment:
This is duplicated with https://github.com/nexB/vulnerablecode/pull/278/files#diff-616b86ac339cf22cb94532d0b2c631ff58d97935da99e6b9ac8fd95569ab2d50R56

Reply:
What's duplicated ? The link is pointing to another unrelated comment.